docs/explanations/bootstrapping.md

```diff
@@ -57,7 +57,7 @@ log-level                                          | debug
 log-level.app                                      | info                                | String  | Match: default                 | Source: remote_cdn_api
 log-level.app.controllers.documentation_controller | debug                               | String  | Match: default                 | Source: /Users/user/.prefab.overrides.config.yaml
 log-level.google.apis.core.http_command            | info                                | String  | Match: default                 | Source: ./.prefab.default.config.yaml
-log-level.prefab                                   | debug                               | String  | Match: default                 | Source: /Users/user/.prefab.overrides.config.yaml
```

Comment:
The source here is `.prefab.overrides.config.yaml`, but the docs in the paragraph above say to set this in `.prefab.default.config.yaml`. I'm not sure which one is more correct, but these two entries should probably agree.

Reply:
Oooh, good catch. I'll update for consistency. Thank you